_data/SyaamantakDas/syaamantakdas.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Syaamantak Das
+institution: Indian Institute of Technology Kharagpur
+profile_pic: syaamantakdas.jpg

```

Comment:
Incorrect file name, please do one of the following:
1. Change `syaamantakdas.jpg` to `syaamantakdas (2).jpg`.
2. Rename `syaamantakdas (2).jpg` to `syaamantakdas.jpg`.
I'd suggest you rename the file!

Reply:
Done. Please check.